driver-core/src/main/com/mongodb/connection/ProtocolHelper.java

```diff
@@ -260,12 +260,11 @@ static void sendCommandStartedEvent(final RequestMessage message, final String d
     }
 
     static void sendCommandSucceededEvent(final RequestMessage message, final String commandName, final BsonDocument response,
-                                          final ConnectionDescription connectionDescription, final long startTimeNanos,
+                                          final ConnectionDescription connectionDescription, final long elapsedTimeNanos,
```

Comment:
Move the calculation of elapsedTimeNanos to the caller to that the same time can be logged in LoggingCommandEventSender.